settings.py

```diff
@@ -126,3 +133,12 @@
 #     "password ": "chaos",
 #     "db ": "db"
 # }
+
+SOURCE_PARTS_NAME = {
+    "system": ["ansible/*", "etc/*", "memoize/*", "puppet/*", "startup.d/*", "chaos_wrapper.sh"],
+    "core": ["cron/*", "chaos.py"],
```

Comment:
Everything is possible. I use `fnmatch`
I worked on feature, that why this is in settings o/

Suggest me change you want :)